packages/react/src/components/Button/ButtonGroup.tsx

```diff
@@ -107,6 +107,6 @@ ButtonGroup.create = createShorthandFactory({
 })
 
 /**
- * A button group presents multiple related actions.
+ * A ButtonGroup represents multiple related actions in a horizontal or vertical group.
```

Comment:
would avoid `horizontal or vertical` as the `ButtonGroup` do not even have the `vertical` prop at this moment

Reply:
removed